coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -57,11 +57,27 @@ def apply(self, result, original_file_dict, file_diff_dict, language: str,
         filename = source_range.file
 
         ignore_diff = Diff(original_file_dict[filename])
-        ignore_diff.change_line(
-            source_range.start.line,
-            original_file_dict[filename][source_range.start.line-1],
-            original_file_dict[filename][source_range.start.line-1].rstrip() +
-            '  ' + ignore_comment)
+
+        if 'Ignore' in str(original_file_dict[filename][
+                    source_range.start.line-1].rstrip()) and 'Bear' in str(
+                    original_file_dict[filename][
+                        source_range.start.line-1].rstrip()):
+            ignore_comment = ignore_comment[
+                ignore_comment.index('Ignore'):].replace('Ignore', ' and')
```

Comment:
don't use `and`, use `comma` for now as it seems like `Processing.py` is only inferring comma separated values, line 476
```
return list(StringConverter(toignore, list_delimiters=', '))
```

Reply:
@ishanSrt both 'and' as well as ``comma`` working fine, let me ask @Makman2 regarding this

Reply:
I used 'and' because it's mentioned in the user's tutorial also.

Reply:
Sorry just checked https://docs.python.org/3/library/stdtypes.html & https://gitlab.com/coala/coala-utils/blob/master/coala_utils/string_processing/StringConverter.py . The strings will be treated as an iterable here.

Reply:
So both 'and' and ``comma`` will work right?

Reply:
Yes, they were not working for me earlier because of some changes I made on my branch 😀

Reply:
Sorry, later. I was solving an issue and saw that `and` wasn't working for me which reminded me that I had suggested the usage of `and` on this issue. So I asked to go for commas.

Reply:
@ishanSrt are you absolutely sure that 'and' wasn't working, please give the content in the file as well as the result.

Reply:
it wasn't working because of the changes I had made. its working on the master branch

Reply:
Okay, so 'and' will do right?

Reply:
yes

Reply:
Great, please review again and acknowledge, if satisfied. Thanks for the review. :smiley:

Reply:
dont you need to replace `Ignore` eith `and` and not with ` and`?

Reply:
@newbazz , either I have to put space before ``and`` ( as I did ) or I have to put space in ``ignore_diff.change_line`` ( after ``new_line`` and before ``ignore_comment`` ), both will work as ``Ignore abcBear and xyzBear``.There must be a space between ``abcBear`` and ``and``. :smile:

Reply:
just add comment of what this code does ;P

Reply:
@newbazz Do I have to add the comment in the code? I have already added comment in the test case though.

Reply:
In the code ;)

Reply:
@newbazz, changes made :smile: . Thanks for the review.